src/pages/myNotes/myNotesDesctiption.jsx

```diff
@@ -0,0 +1,10 @@
+import PropTypes from 'prop-types';
+
+const MyNoteDesctiption = ({ activeNote }) => {
```

Comment:
We should have only one `return` statement per component. Maybe it will be better to write something like this?
```
const MyNoteDesctiption = ({ activeNote }) => (
  <>
  { activeNote ? <p>{activeNote.description}</p> : <div className="no-active-note"  style={{ flex: "3" }}>No Active Note</div> }
  </>
)
```